test/com/collections/MethodsTest.java

```diff
@@ -38,6 +39,60 @@ void compare() {
         firstSet.add(4);
 
         Set<Integer> secondSet = new HashSet<>(firstSet);
-        assertTrue((methods.compare(firstSet,secondSet)));
+        assertTrue((methods.compare(firstSet, secondSet)));
+    }
+
+    @Test
+    void createPriorityQueue() {
+        Employee employee1 = new Employee("Kazek", 2000);
+        Employee employee2 = new Employee("Dzol", 2500);
+        Employee employee3 = new Employee("Boyd", 3000);
+        Employee employee4 = new Employee("John", 3500);
+        Employee employee5 = new Employee("Peter Long", 4000);
+        Employee employee6 = new Employee("Albert",1023);
+        Employee employee7 = new Employee("Zack", 8000);
+        Employee employee8 = new Employee("Bill", 3450);
+
+        List<Employee> list = new ArrayList<>();
+        list.add(employee1);
+        list.add(employee2);
+        list.add(employee3);
+        list.add(employee4);
+        list.add(employee5);
+        list.add(employee6);
+        list.add(employee7);
+        list.add(employee8);
+
+        PriorityQueue<Employee> queue = methods.createPriorityQueue(list);
+
+        System.out.println("priority queue: " + queue);
```

Comment:
When you printing queue elements order can be different from result when you're taking elements from the queue.
Let's try to check if next element on queue exists and if yes take it from the queue (use methods from PriorityQueue class to do that). Do it for all elements.